src/app-layout/visual-refresh-toolbar/drawer/local-drawer.tsx

```diff
@@ -20,9 +20,13 @@ import styles from './styles.css.js';
 
 interface AppLayoutDrawerImplementationProps {
   appLayoutInternals: AppLayoutInternals;
```

Comment:
Not a blocker for this PR, but there is an API design issue. We are not supposed to pass down `appLayoutInternals` more than one level deep. Compare with this example: https://github.com/cloudscape-design/components/blob/e57c945bbfb8b04840ec4af904e72e86f6866378/src/app-layout/visual-refresh-toolbar/toolbar/index.tsx#L213-L227

When passing `appLayoutInternals` all over the place, it makes refactoring impossible, because you cannot track what is used where and how